packages/storybook/src/executors/models.ts

```diff
@@ -6,14 +6,35 @@ export interface StorybookConfig {
 }
 
 export interface CommonNxStorybookConfig {
-  uiFramework:
-    | '@storybook/angular'
-    | '@storybook/react'
-    | '@storybook/html'
-    | '@storybook/web-components'
-    | '@storybook/vue'
-    | '@storybook/vue3'
-    | '@storybook/svelte'
-    | '@storybook/react-native';
+  uiFramework?: UiFramework;
+  uiFramework7?: UiFramework7;
   config: StorybookConfig;
 }
+
+export type UiFramework7 =
+  | '@storybook/angular'
+  | '@storybook/html-webpack5'
+  | '@storybook/next'
```

Comment:
@mandarini Sorry, I was wrong. It has to be @storybook/nextjs instead of @storybook/next